jubjub/src/lib.rs

```diff
@@ -140,6 +162,18 @@ impl PartialEq for ExtendedPoint {
     }
 }
 
+impl Sum for ExtendedPoint {
+    fn sum<I: Iterator<Item = Self>>(iter: I) -> Self {
+        iter.fold(Self::identity(), Add::add)
+    }
+}
+
+impl<'r> Sum<&'r ExtendedPoint> for ExtendedPoint {
```

Comment:
If we can't use a blanket trait for these, we should at least make a macro for them.

Reply:
There's already a set of common macros that are planned to be extracted out; we can do this at the same time.